help/destinations/catalog/crm/zendesk.md

```diff
@@ -0,0 +1,158 @@
+1. In the **[!UICONTROL Select source field]** window, choose the **[!UICONTROL Select attributes]** category and select the XDM attribute or choose the **[!UICONTROL Select identity namespace]** and select an identity.
+1. In the **[!UICONTROL Select target field]** window, choose the **[!UICONTROL Select identity namespace]** and select an identity or choose **[!UICONTROL Select custom attributes]** category and select an attribute as needed.
+    * Repeat these steps to add the following mappings between your XDM profile schema and your [!DNL Zendesk] instance:
+        |Source Field|Target Field| Mandatory|
+        |---|---|---|
+        |`xdm: person.name.lastName`|`Attribute: last_name`| Yes, Note you can also use `name`. |
+        |`IdentityMap: Email`|`Identity: email`| Yes |
+
+    * An example using these mappings is shown below:
+    ![Platform UI screenshot example with attribute mappings.](../../assets/catalog/crm/zendesk/mappings.png)
+
+>[!IMPORTANT]
+>
+>You must have a mapping for *Last Name* or *Name* otherwise the [!DNL Zendesk] API does not respond with an error, and attribute value ignored.
+
+When you are finished providing the mappings for your destination connection, select **[!UICONTROL Next]**.
+
+### Schedule segment export and example {#schedule-segment-export-example}
+
+In the [[!UICONTROL Schedule segment export]](/help/destinations/ui/activate-segment-streaming-destinations.md#scheduling) step of the activation workflow, you must manually map Platform segments to the custom field attribute in [!DNL Zendesk].
+
+To do this, select each segment, then enter the corresponding custom field attribute from [!DNL Zendesk] in the **[!UICONTROL Mapping ID]** field.
+
+An example is shown below:
+![Platform UI screenshot example showing Schedule segment export.](../../assets/catalog/crm/zendesk/schedule-segment-export.png)
+
+## Validate data export {#exported-data}
+
+To validate that you have correctly set up the destination, follow the steps below:
+
+1. Select **[!UICONTROL Destinations]** > **[!UICONTROL Browse]** and navigate to the list of destinations.
+1. Next, select the destination and switch to the **[!UICONTROL Activation data]** tab, then select a segment name.
+![Platform UI screenshot example showing Destinations Activation Data.](../../assets/catalog/crm/zendesk/destinations-activation-data.png)
+
+1. Monitor the segment summary and ensure that the count of profiles corresponds to the count within the segment.
+![Platform UI screenshot example showing Segment.](../../assets/catalog/crm/zendesk/segment.png)
+
+1. Log in to the [!DNL Zendesk] website, then navigate to the **[!UICONTROL Contacts]** page to check if the profiles from the segment have been added. This list can be configured to display columns for the additional fields created with the segment Mapping ID and segment statuses.
```

Comment:
Mapping ID should be enclosed in UICONTROL tags.

Reply:
Thanks